src/engine/sparqlExpressions/PrefilterExpressionIndex.h

```diff
@@ -261,6 +301,9 @@ class RelationalExpression : public PrefilterExpression {
   std::string asString(size_t depth) const override;
 
  private:
+  // `PrefixRegexExpression::evaluateMirroredImpl` potentially requires direct
+  // access to the `evaluateImpl` below.
+  friend class PrefixRegexExpression;
```

Comment:
As that function doesn't exist anymore, do we still need the `friend` + comment.